docs/rst/community/index.rst

```diff
@@ -0,0 +1,182 @@
+Community Information & Contributing
+====================================
+
+Ansible Container is an open source project designed to bring together developers of all kinds to collaborate on building a better way to build, test and deploy containers. 
+
+Should you wish to get more involved – whether in terms of just asking a question, helping other users, introducing new people to Ansible Container, or helping build the 
+software or documentation, we welcome your contributions:
+
+.. contents:: Topics
+
+Users
+`````
+
+I've got a question
+-------------------
+
+We're happy to help!
+
+Questions are best asked on the `Ansible Container Mailing List <https://groups.google.com/forum/?hl=en-GB#!forum/ansible-container>`_.
+
+When asking a question please be sure to share any relevant commands you ran, output, the version of Ansible Container you are using, etc. Where needed, 
+link to gists or github repos to show examples, rather than sending attachments to the list.
+
+Alternatively, you can also join our IRC channel - #ansible-container on irc.freenode.net.
+
+I’d Like To Keep Up With Release Announcements
+----------------------------------------------
+Release announcements will be posted to the `Ansible Container Mailing List <https://groups.google.com/forum/?hl=en-GB#!forum/ansible-container>`_. 
+
+
+I’d Like To Help Share and Promote Ansible Container
+----------------------------------------------------
+You can help share Ansible with others by telling friends and colleagues, writing a blog post, or presenting at user groups (like DevOps groups or the local LUG).
+
+You are also welcome to share slides on speakerdeck, sign up for a free account and tag it “Ansible Container”. On Twitter, you can also share things with #ansible-container 
```

Comment:
Hashtags may not contain hyphens. Link to speakerdeck, and replace the comma after the word "speakerdeck" with a colon. Link to `@ansible` with Twitter intent API link.